src/community/ogcapi/ogcapi-core/src/main/resources/org/geoserver/ogcapi/common-header.ftl

```diff
@@ -2,6 +2,7 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
+      <meta http-equiv="content-type" content="text/html; charset=${.output_encoding}">
```

Comment:
There is a fixed ``<meta charset="UTF-8">`` [a few lines below](https://github.com/geoserver/geoserver/blob/main/src/community/ogcapi/ogcapi-core/src/main/resources/org/geoserver/ogcapi/common-header.ftl#L10
)... not sure what happens if there are two, but if you did manual testing, that could have interfered.